swift/StableDiffusion/pipeline/CGImage+vImage.swift

```diff
@@ -105,6 +105,19 @@ extension CGImage {
             
             vImageConvert_ARGB8888toPlanarF(&mediumDesination, &destinationA, &destinationR, &destinationG, &destinationB, &maxFloat, &minFloat, .zero)
            
+            let destAPtr = destinationA.data.assumingMemoryBound(to: Float.self)
+            let destRPtr = destinationR.data.assumingMemoryBound(to: Float.self)
+            let destGPtr = destinationG.data.assumingMemoryBound(to: Float.self)
+            let destBPtr = destinationB.data.assumingMemoryBound(to: Float.self)
+
+            for i in 0..<Int(width) * Int(height) {
+                if destAPtr.advanced(by: i).pointee < 1 {
```

Comment:
Can we get away with ==0 instead of <1?

Reply:
Definitely, I just pushed the change.

We're on uncharted territory, as the Python world used a separate mask image, where dark pixels (luminance under 0.5) were masked out. But I think most image editing tools, including Preview, will just erase to 0 anyway 🙂